src/ext.test.ts

```diff
@@ -2,143 +2,110 @@ import { FileKind } from './data';
 import { fromExtension, getExtensions } from './ext';
 
 describe('extension', () => {
+  const getTestCases = (extensions: string[], type: string) => {
+    const testCases: Array<[string, string]> = [];
+    extensions.forEach((ext) => {
+      testCases.push([ext, type]);
+      testCases.push([ext?.toUpperCase(), type]);
```

Comment:
Please don't hide this here. There should be an explicit test to describe the behaviour that uppercase extensions are supported.